_pages/en/capabilities.md

```diff
@@ -386,82 +386,75 @@ InfoSec supplies development teams and operational teams with an ample number of
+  c. Undecided  
+  d. Disagree  
+  e. Strongly Disagree  
+
+1-7 (very frequently to never)  
+Ideal: Very Frequently  
+
+- Does monitoring in your team give rapid feedback which helps quickly find and fix problems early in the software development lifecycle?  
+  a. Very Frequently (Ideal)  
+  b. Frequently  
+  c. Occasionally  
+  d. Rarely  
+  e. Very Rarely  
+  f. Never  
+
+- In my team the knowledge gained from application monitoring is shared across organization to help people and systems improve  
+  a. Very Frequently (Ideal)  
+  b. Frequently  
+  c. Occasionally  
+  d. Rarely  
+  e. Very Rarely  
+  f. Never  
+
+- In my team collected data is always accessible to different audiences (visibility into development work, testing, QA, and IT operations) to help them make decisions  
+  a. Strongly Agree (Ideal)  
+  b. Agree  
+  c. Undecided  
+  d. Disagree  
+  e. Strongly Disagree  
+
+- When monitoring a system,  my team is ...  
+  a. Monitoring reactively. (Only getting alerted when the system goes down, but not using monitoring data to help alert when the system approaches critical thresholds)  
+  b. Monitoring too small a scope. (Monitoring one or two areas rather than the full software development and delivery pipeline)  
+  c. Focusing on local optimizations. (Focusing on reducing the response time for one service's storage needs without evaluating whether the broader infrastructure could also benefit from the same improvement)  
+  d. Monitoring everything  
+  e. Taking a thoughtful approach to monitoring which usually draws attention to key areas (Ideal)  
+  f. We do not do system  monitoring  
 
 - How fast your team reacts to alerts when the system approaches critical thresholds or goes down?  
-   a. When the threshold reaches 25%  
-   b. When the threshold reaches 50%  
-   c. When the threshold reaches 75%  
-   d. When the threshold reaches 85%  
-   e. When the threshold reaches 95%  
-
-- How is the monitoring process executed in your team?  
-   a. Monitoring one or two areas at a time  
-   b. Monitoring the full software development  
-   c. We are not monitoring  
-
-- Which tool do you use to share collected data in your team?  
-   a. Google docs  
-   b. Sharepoint  
-   c. Email  
-   d. Office 365  
-   e. Slack  
-   f. None of the above/not applicable  
-
-- What has the most impact to make a business decision in your team?  
-   a. Data from application performance monitoring tools  
-   b. Data from infrastructure monitoring tool  
-   c. Outside perspective  
-   d. Previous decision mistakes  
-   e. Your gut instinct  
-   f. None of the above/not applicable  
-
-- How usually  accurate and relevant  is the data to your goal of the team?  
-   a. Very Frequently  
-   b. Frequently  
-   c. Occasionally  
-   d. Rarely  
-   e. Very Rarely  
-   f. Never  
+  Percentage slider  
+  Ideal: up to 25%  
+
+- How usually accurate and relevant is the data to your goal of the team?  
```

Comment:
- How usually accurate and relevant is the data to your goal of the team?  
===>
- Generally, how accurate and relevant is the data to the team?